types/obelisk.js/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for obelisk.js v1.1.0
+// Type definitions for obelisk.js v1.2.1
 // Project: https://github.com/nosir/obelisk.js/
 // Definitions by: Brian Drupieski <https://github.com/bdrupieski>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
+export = obelisk;
```

Comment:
Instead of making it exported this way, you should do the following:
```ts
export as namespace obelisk; // this makes the module available globally

export var version: string; // remove the declare namespace so all exports are at the top level
```